src/common/components/layouts/pages/inspect/validationErrorStatus.json

```diff
@@ -211,6 +211,101 @@
         "STATUS": "warning",
         "CATEGORY": "structure",
         "RATIONALE": "Lists should not be misused to represent tables."
+      },
+      "27": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Multiple paragraphs should be tagged as such. Line breaks should not be used to delimit paragraphs."
+      },
+      "28": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of content items should be tagged as such."
+      },
+      "29": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of content items should not be empty."
+      },
+      "30": {
+        "STATUS": "warning",
+        "CATEGORY": "code",
+        "RATIONALE": "Table of content items should point somwhere in the document."
```

Comment:
somwhere -> somewhere.
It's rule not about link, it's about toci page label. Maybe we need to write a more understandable message